sdk/src/types/Tranche.ts

```diff
@@ -0,0 +1,34 @@
+import * as anchor from "@project-serum/anchor";
+
+
+export interface LastUpdateType {
```

Comment:
I don't get why we need both an interface and a class for the same type, can't we use just the class object?

Reply:
I tried that but there are some nested fields in TrancheConfig which are of type anchor.BN such as LastUpdate. For those I took them as input in create function and converted them to number there. For that input argument type  I created a separate interface with anchor.BN.

Reply:
mmh ok, I see. maybe we can structure our code with no anchor related stuff (i.e. `anchor.BN`) and when is time to convert fetched class from the chain to our classes we convert any non standard data type to a standard one. You can use this pattern as a reference (should have access to pvt repo):
https://github.com/vyper-protocol/vyper-vaults/blob/main/app/sdk/src/VaultData.ts